ports/stm32/boards/ARDUINO_PORTENTA_H7/mpconfigboard.h

```diff
@@ -206,6 +206,11 @@ extern struct _spi_bdev_t spi_bdev;
 #define MICROPY_HW_SDCARD_D3        (pin_B4)
 #define MICROPY_HW_SDCARD_MOUNT_AT_BOOT (0)
 
+// FDCAN bus
+#define MICROPY_HW_CAN1_NAME                "FDCAN1"
+#define MICROPY_HW_CAN1_TX                  (pin_D1)
+#define MICROPY_HW_CAN1_RX                  (pin_D0)
```

Comment:
```suggestion
#define MICROPY_HW_CAN1_NAME                "FDCAN1"
#define MICROPY_HW_CAN1_TX                  (pin_H13)
#define MICROPY_HW_CAN1_RX                  (pin_B8)
``` 

Those should be `H13` and `B8`.

Reply:
I used the processor pin designators, but can change to the board designators and re-submit.  If I understand correctly, though, `pin_H13` and `pin_B8` would end up using the CPU designators as well. Should I use the following, instead?

```
#define MICROPY_HW_CAN1_TX                  (pyb_pin_H13)
#define MICROPY_HW_CAN1_RX                  (pyb_pin_B8)
```

And then add the following to `pins.csv` (per the comments at the top of `make_pins.py`)?
```
B8,PD0
H13,PD1
```

Reply:
Those are processor pins and they are the ones connected to the CAN that's broken out on the B2B header, D0 and D1 are used for FMC.. There's no need to change anything else.